packages/cosmwasm-stargate/src/queries/wasm.spec.ts

```diff
@@ -386,15 +394,13 @@ describe("WasmExtension", () => {
         codeId = Number.parseInt(codeIdAttr.value, 10);
         expect(codeId).toBeGreaterThanOrEqual(1);
         expect(codeId).toBeLessThanOrEqual(200);
-        expect(result.data!.length).toEqual(1);
-        console.log(`Raw Store Data: [${result.data![0].data}]`);
-        console.log(`Code ID From events: ${codeId}`);
-        expect({ ...result.data![0] }).toEqual({
-          msgType: "store-code",
-          // TODO: protobuf de/encode here `{codeId: codeId}`
-          // https://github.com/CosmWasm/wasmd/blob/5f8c246d25e8be640fb401fda4bbf82db37e9e90/x/wasm/internal/types/tx.proto#L41-L45
-          data: toAscii(`${codeId}`),
-        });
+
+        assertDefined(result.data);
+        const msgData = fromOneElementArray(result.data);
+        expect(msgData.msgType).toEqual("store-code");
+        expect(MsgStoreCodeResponse.decode(msgData.data!)).toEqual(
```

Comment:
Looks nice.

can you just use `MsgStoreCodeResponse.create({ codeId })` below? I thought it would auto-covert number to long in create (and we will not have anything over 2^53

Reply:
> can you just use `MsgStoreCodeResponse.create({ codeId })` below? I thought it would auto-covert number to long in create (and we will not have anything over 2^53

We changed this by passing `--force-long` to `pbjs`. This way we get consistent behaviour across all systems and values.